app/views/vehicles/new.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= f.input :description %>
   <%= f.input :location %>
   <%= f.input :price %>
-  <%= f.input :category %>
-  <%= f.input :photos, as: :file, input_html: { multiple: true } %>
```

Comment:
we really need that line for the image upload

Reply:
Ah shit very sorry! I only wanted to change the category line, good catch